<ui version="4.0" >
 <class>SourceConcentrationDialog</class>
 <widget class="QDialog" name="SourceConcentrationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Source Chemical Concentration Levels</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>550</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="_chemicalTable" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>60</y>
     <width>731</width>
     <height>441</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>7</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>53</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Sources</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>30</y>
     <width>78</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Contaminants</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sourceDropDown" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QtTools::ColorButton" name="color" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>180</y>
     <width>44</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>180</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceName" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>100</y>
     <width>33</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton" >
   <property name="geometry" >
    <rect>
     <x>550</x>
     <y>550</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header>QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SourceConcentrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SourceConcentrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
